posthog/warehouse/api/saved_query.py

```diff
@@ -104,6 +115,14 @@ class Meta:
             "soft_update": {"write_only": True},
         }
 
+    def get_snapshot_config(self, view: DataWarehouseSavedQuery) -> DataWarehouseSnapshotConfig | None:
+        try:
+            return view.datawarehousesnapshotconfig.config
```

Comment:
Will this create a n+1 issue? Are we pre-loading `datawarehousesnapshotconfig` in the right places?